src/main/java/net/io/kino/service/EmailSender.java

```diff
@@ -0,0 +1,7 @@
+package net.io.kino.service;
+
+import net.io.kino.model.Order;
+
+public interface EmailSender {
+    void sendEmail(String to, Order order);
```

Comment:
Jak tak myślę to w sumie powielane są informacje, bo Order ma PersonalDetails, które ma email